test/integration/hooks/associations.test.js

```diff
@@ -6,7 +6,8 @@ var chai = require('chai')
   , Support = require(__dirname + '/../support')
   , DataTypes = require(__dirname + '/../../../lib/data-types')
   , sinon = require('sinon')
-  , dialect = Support.getTestDialect();
+  , dialect = Support.getTestDialect()
+  , Promise = require('bluebird');
 
 describe(Support.getTestDialectTeaser('Hooks'), function() {
```

Comment:
Almost all `function` expression here are missing space :laughing:, We should fix this formatting issue too